src/WROVER_KIT_LCD.cpp

```diff
@@ -114,6 +114,50 @@ uint32_t WROVER_KIT_LCD::readId() {
     return r;
 }
 
+
+uint16_t WROVER_KIT_LCD::readPixels(uint16_t *colors, uint32_t len) {
```

Comment:
you are returning 16 bits from 32bit value

Reply:
oh bummer, fixed that quick :D